src/crimson/osd/osd_operations/logmissing_request_reply.cc

```diff
@@ -61,8 +67,17 @@ seastar::future<> LogMissingRequestReply::with_pg(
 
   IRef ref = this;
   return interruptor::with_interruption([this, pg] {
-    return pg->do_update_log_missing_reply(std::move(req));
-  }, [ref](std::exception_ptr) { return seastar::now(); }, pg);
+    return pg->do_update_log_missing_reply(std::move(req)
+    ).then_interruptible([this] {
+      logger().debug("{}: complete", *this);
+      return handle.complete();
+    });
+  }, [](std::exception_ptr) {
+    return seastar::now();
+  }, pg).finally([this, ref] {
+    logger().debug("{}: exit", *this);
```

Comment:
@athanatos FYI, the ref-counter issue of https://github.com/ceph/ceph/pull/53537#issuecomment-1761029984 turns out to because that this line is trying to print `this` op without a valid req (moved away at line 70 above), causing seg fault according to https://github.com/ceph/ceph/blob/9587ffdfc8c52bcf06abbe89d2f527c434a3e9f5/src/crimson/osd/osd_operations/logmissing_request_reply.cc#L28-L34